torch/csrc/autograd/variable.h

```diff
@@ -22,7 +22,7 @@ struct Variable : std::enable_shared_from_this<Variable> {
     SavedVariable(const Variable& variable, Function* saved_for)
       : data(variable.data->clone_shallow())
       , grad_accumulator(variable.grad_accumulator)
-      , version(std::move(variable.version_counter->new_saved_ref()))
+      , version(variable.version_counter->new_saved_ref())
```

Comment:
Why isn't the `std::move` needed here? `new_saved_ref` returns a `std::unique_ptr` no?